OpenXmlPowerTools/DocumentBuilder.cs

```diff
@@ -221,6 +252,10 @@ private class Atbid
             public int Div;
         }
 
+        public const string Yes = "yes";
```

Comment:
Do these need to be public? I tend to prefer to keep stuff private unless necessary so that they are not set in stone for backwards compatibility.